tests/foreman/ui/test_settings.py

```diff
@@ -125,6 +126,31 @@ def test_positive_update_restrict_composite_view(session, set_original_property_
                 assert lce.name in result['Environments']
 
 
+@skip_if_bug_open('bugzilla', 1677282)
+@tier2
+def test_positive_httpd_proxy_url_update(session, set_original_property_value):
+    """Update the http_proxy_url should pass successfully.
+
+    :id: 593eb8e1-16dd-486b-a760-47b8fdf4dcb9
+
+    :expectedresults: http_proxy_url updated successfully.
+
+    :BZ: 1677282
+
+    :CaseImportance: Medium
+
+    """
+    property_name = 'http_proxy'
+    with session:
+        set_original_property_value(property_name)
+        param_value = gen_url()
+        session.settings.update(
+            'name = {}'.format(property_name),
+            param_value
```

Comment:
```suggestion
            gen_url()
```
Can we directly call as above ?

Reply:
hmm good call